sos/report/plugins/ceph_common.py

```diff
@@ -41,57 +42,38 @@ class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
 
     # This check will enable the plugin regardless of being
     # containerized or not
-    files = (
-        '/etc/ceph/ceph.conf',
-    )
+    files = ('/etc/ceph/ceph.conf',)
 
     def setup(self):
         all_logs = self.get_option("all_logs")
 
         self.add_file_tags({
             '.*/ceph.conf': 'ceph_conf',
-            '/var/log/ceph/ceph.log.*': 'ceph_log',
-            '/var/log/ceph/ceph-osd.*.log': 'ceph_osd_log'
+            '/var/log/ceph/ceph.log*': 'ceph_log',
         })
 
         if not all_logs:
-            self.add_copy_spec([
-                "/var/log/ceph/*.log",
-                "/var/log/radosgw/*.log",
-                "/var/log/calamari/*.log"
-            ])
+            self.add_copy_spec("/var/log/calamari/*.log",)
         else:
-            self.add_copy_spec([
-                "/var/log/ceph/",
-                "/var/log/calamari",
-                "/var/log/radosgw"
-            ])
+            self.add_copy_spec("/var/log/calamari",)
 
         self.add_copy_spec([
+            "/var/log/ceph/ceph.audit.log*",
+            "/var/log/calamari/*.log",
             "/etc/ceph/",
             "/etc/calamari/",
-            "/var/lib/ceph/",
-            "/run/ceph/"
+            "/var/lib/ceph/tmp/",
         ])
 
         self.add_cmd_output([
-            "ceph mon stat",
-            "ceph mon_status",
             "ceph quorum_status",
```

Comment:
Does the ceph_common meant to be run on all different nodes (OSD, mon, mgr, etc)?

If so, most of these commands probably need to go elsewhere such as mon & mgr.

Reply:
yes common will run on all nodes. Yes, ack.